File: eHairdressers_admin/flutter_ehairdressers/lib/models/appointment.dart
import 'package:json_annotation/json_annotation.dart';

part 'appointment.g.dart';

@JsonSerializable()
class Appointment {
  int? appointmentId;
  int? employeeId;
  int? userId;
  int? serviceId;
  String? comment;
  String? employeeName;
  String? username;
  String? serviceName;

  Appointment(this.appointmentId, this.employeeId, this.userId, this.serviceId,
      this.comment, this.employeeName, this.username, this.serviceName);

  factory Appointment.fromJson(Map<String, dynamic> json) =>
      _$AppointmentFromJson(json);

  Map<String, dynamic> toJson() => _$AppointmentToJson(this);
}

File: eHairdressers/eHairdressers.Model/Appointment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eHairdressers.Model
{
    public class Appointment
    {
        public int AppointmentId { get; set; }
        public int EmployeeId { get; set; }
        public string EmployeeName { get; set; }
        public int UserId { get; set; }
        public string Username { get; set; }
        public int ServiceId { get; set; }
        public string ServiceName { get; set; }
        public string AppointmentDate { get; set; }
        public string AppointmentTime { get; set; }
        public bool Approved { get; set; }
        public string Comment { get; set; }

    
    }
}
